include/mender-client.h

```diff
@@ -64,7 +64,7 @@ typedef struct {
     mender_err_t (*network_release)(void);                                 /**< Invoked when mender-client releases access to the network */
     mender_err_t (*deployment_status)(mender_deployment_status_t, char *); /**< Invoked on transition changes to inform of the new deployment status */
     mender_err_t (*restart)(void);                                         /**< Invoked to restart the device */
-    mender_err_t (*get_identity)(mender_identity_t **identity);            /**< Invoked to retrieve identity */
+    mender_err_t (*get_identity)(const mender_identity_t **identity);      /**< Invoked to retrieve identity */
```

Comment:
This change is good. Let's be aware that it will break `mender-mcu-integration` build (right?).

Reply:
yup, I'll adapt that.

Reply:
Alright, you adapt it then. :wink: